cmd/argocd/commands/login.go

```diff
@@ -323,9 +323,10 @@ func oauth2Login(
 		log.Fatalf("Unsupported grant type: %v", grantType)
 	}
 	fmt.Printf("Performing %s flow login: %s\n", grantType, url)
+	if err := open.Start(url); err != nil {
```

Comment:
Maybe still log the error, just in case it's something the user will want to diagnose/resolve?